docs/guides/getting-started/fr/getting-started-guide.md

```diff
@@ -0,0 +1,173 @@
+# Commencer avec Omnivore
+
+Omnivore est une **application de lecture ultérieure** qui vous permet de sauvegarder et d'organiser tout ce que vous lisez en ligne.
+
+Ce guide vous montrera comment utiliser les fonctions de base et les fonctionnalités avancées d'Omnivore, divisées en quatre activités principales:
+
+
+* Sauvegarde
+* Lecture
+* Organisation
+* Intégrations
+
+La **bibliothèque** est le centre de votre expérience Omnivore, où vous pouvez accéder rapidement à tous les liens que vous avez enregistrés. Les liens enregistrés restent dans votre bibliothèque à jamais, à moins que vous ne les supprimiez.
+
+
+## Sauvegarde
+
+
+Il existe cinq façons de sauvegarder des liens vers des pages ou des articles que vous souhaitez lire plus tard:
+
+
+
+* Sauvegarde depuis votre bibliothèque Omnivore
+* Sauvegarde depuis un navigateur
+* Sauvegarde depuis un téléphone ou une tablette (iOS ou Android)
+* Abonnements à des newsletters via e-mail
+* Sauvegarde de PDF à partir d'un Mac
+
+
+### Saving from Your Omnivore Library
+1. Dans le coin supérieur droit de votre bibliothèque, appuyez sur le bouton **Ajouter un lien**.
```

Comment:
@laurentkempe sorry for the late reply, yeah I suspect we should keep UI elements in English as it might be some time until we can get all messages translated (especially on web where we don't have a framework for translation like iOS).

Reply:
Ok, no worries, we share the same view on that then. I will adapt the doc again, then make final review and it should be good. I will let you know.